cs/ccxt/base/Exchange.Generic.cs

```diff
@@ -17,6 +17,8 @@ public List<object> sortBy(object array, object value1, object desc2 = null, obj
         if (value1.GetType() == typeof(string))
         {
             var sortedList2 = list.OrderBy(x => ((dict)x)[(string)value1]).ToList();
+            if (desc)
```

Comment:
we were missing `descenging` argument here, like we have in "else" clause (few lines below)